src/MizarWidgets/SamplingWithFrameTrackOutputWidget.ui

```diff
@@ -14,6 +14,13 @@
    <string>Form</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLineEdit" name="filter_line_">
+     <property name="placeholderText">
+      <string>Filter by functions name</string>
```

Comment:
functions or function? Without the s sounds better to me in my head, but I am no expert